06-Customizing AzTS for your org/Extending AzTS/UpdateControlMetadata.md

```diff
@@ -1,8 +1,8 @@
-## Update existig control metadata
-Control metadata like Display Name, Category, Tags, Description , Recommendations can be updated using Control Metadata Editor Tool(CMET). Please follow the below mentioned steps to update control metadata:
+## Update existing control metadata
+Control metadata like Display Name, Category, Tags, Description, Recommendations can be updated using Control Metadata Editor Tool(CMET). Please follow the below mentioned steps to update control metadata:
```

Comment:
To update control metadata, including fields like Display Name, Category, Tags, Description, and Recommendations, you can utilize the Control Metadata Editor Tool (CMET). Follow the steps below to update control metadata:

Reply:
Updated